docs/integrations/microsoft-azure/opentelemetry/sql-server-opentelemetry.md

```diff
@@ -0,0 +1,174 @@
+---
+id: sql-server-opentelemetry
+title: Microsoft SQL Server - OpenTelemetry Collector
+sidebar_label: Microsoft SQL Server - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Microsoft SQL Server.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/sql.png')} alt="thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for Microsoft SQL Server is a logs based app that provides insight into your SQL server. The App consists of predefined Dashboards, providing visibility into your environment for real-time or historical analysis on backup, restore mirroring, general health and operations of your system.
+
+This App has been tested with following SQL Server versions:
+
+-   Microsoft SQL Server 2012
+
+SQL Server logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-OpenTelemetry/SQL-Server-Schematics.png')} alt="Redis Logs dashboards" />
+
+## Fields creation in Sumo Logic for SQL Server
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of SQL Server App install if not already present.
+
+**`db.cluster.name`** - User configured. Enter a name to identify this SQL Server cluster. This cluster name will be shown in the Sumo Logic dashboards.
```

Comment:
This field list is getting rendered as a whole paragraph and not seperate points